module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,56 @@
+#include "compression.hpp"
+
+#include <iostream>
+
+
+compressedBitmap compressGrayscale(const image& img) {
+    compressedBitmap cb{};
+
+    for (const auto& row : img) {
+        cb.emplace_back(std::make_pair(0, 0));
```

Comment:
I don't like it. This is hard to read and understand. You will always have at the beginning {0, 0}. I know You will be increment value later, but this is hard to read.
KISS -> Keep it simple :)

Reply:
@WBigaj
And when you are using emplace_back you do not need to call std::make_pair
You can do it like this:
cb.emplace_back(0,0);